### PR TITLE
Bump Hibernate from 3 to 4.

### PR DESCRIPTION
**Pull request by mackos**

### CODE REVIEW

**Comment by mackos** [comment]
@Decker108 : Does this seem reasonable?

File: pom.xml [hunk 1]
```diff
@@ -199,11 +204,6 @@
       <version>3.8.0.GA</version>
     </dependency>
     <dependency>
-      <groupId>org.hibernate</groupId>
-      <artifactId>hibernate-annotations</artifactId>
```
Comment (Decker108) [inline pom.xml]: Why was this removed? Is it included in hibernate 4?
